libs/mock-block-dock/src/version.ts

```diff
@@ -1 +1 @@
-export const MOCK_BLOCK_DOCK_VERSION = "0.1.1";
+export const MOCK_BLOCK_DOCK_VERSION = "0.1.2";
```

Comment:
Does this only happen after a PR or something? I seem to have applied this as a drive-by once or twice as well and I'm unsure if we're just missing it in linting or

Reply:
This is a `codegen` in `mock-block-dock`. I'm not sure why it doesn't get run when the MBD version is _actually_ updated. It seems to just happen in branches afterwards.

Reply:
Is it to do with `changesets` managing our version bumps but probably not committing the results of codegen?

Reply:
Oh, yes, that explains it